timemachine/md/moves.py

```diff
@@ -103,6 +108,31 @@ def n_proposed_by_move(self) -> List[int]:
         return [m._n_proposed for m in self.moves]
 
 
+class Choice(CompoundMove[_State]):
+    """Apply a single move uniformly selected from a list"""
+
+    def __init__(self, moves: Sequence[MonteCarloMove[_State]]):
+        self.moves = moves
+
+    def move(self, x: _State) -> _State:
+        idx = np.random.choice(len(self.moves))
+        chosen_move = self.moves[idx]
+        x = chosen_move.move(x)
+        return x
+
+
+class Sequential(CompoundMove[_State]):
```

Comment:
nit: naming -- `SequenceOfMoves` or similar?

Reply:
Renamed in https://github.com/proteneer/timemachine/pull/1128/commits/543cac9b411362b37946bf85c0c251cc4e7d7711